src/main/java/app/chatbot/Response.java

```diff
@@ -0,0 +1,27 @@
+package app.chatbot;
+
+/**
+ * Helper class for formatting GUI responses.
+ */
+public class Response {
+    private StringBuilder sb;
+
+    public Response() {
+        this.sb = new StringBuilder();
+    }
+
+    public Response(String s) {
```

Comment:
Missing JavaDocs, one-liners should do. 

The same goes for the method below (Without the JavaDoc for `addLine` users don't know that the line is added to the stringbuilder rather than the input string itself)
```
public void addLine(String input) {
```